storage/service_account/hmac/hmac_test.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//:
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hmac
+
+import (
+	"bytes"
+	"context"
+	"os"
+	"testing"
+
+	"cloud.google.com/go/storage"
+)
+
+var ProjectID = os.Getenv("GOOGLE_CLOUD_PROJECT")
```

Comment:
Should get `ProjectID` from `testutil.SystemTest`:

```go
tc := testutil.SystemTest(t)
// Use tc.ProjectID.
```

Reply:
This required passing in projectID as a param to the setup method, but I think that's fine.